spec/creating_post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'capybara/rspec'
+
+feature 'user can create a new post' do
+	scenario "User logs in" do
+		visit '/'
+		fill_in('password', :with => 'rhys@rhys.com')
+		fill_in('password', :with => 'test')
```

Comment:
Each time you write the code to feature test the login, you’re filling in the password field twice, instead of the email and the password fields!